contracts/GovHub.sol

```diff
@@ -19,7 +19,7 @@ contract GovHub is System, IApplication {
 
     event failReasonWithStr(string message);
     event failReasonWithBytes(bytes message);
-    event paramChange(string key, bytes value);
+    event paramChange(string key, bytes value);  // @dev deprecated
 
     struct ParamChangePackage {
```

Comment:
`using RLPDecode for *;` no longer needed

Reply:
removed for these `using`